cmd/backend.go

```diff
@@ -1,18 +1,124 @@
 package main
 
 import (
+	"flag"
+	"fmt"
 	"github.com/grafana/grafana-build/pipelines"
 	"github.com/urfave/cli/v2"
+	"strings"
 )
 
-var TestBackendUnit = &cli.Command{
-	Name:   "test",
-	Action: PipelineAction(pipelines.GrafanaBackendTests),
+type ChoiceFlag struct {
```

Comment:
I would appreciate a comment above here that describes the intention with the ChoiceFlag and how to use it.

```
// ChoiceFlag is a cli.Flag that ...
```